src/views/ConsentContinuationOut.vue

```diff
@@ -403,9 +413,13 @@ export default class ConsentContinuationOut extends Mixins(CommonMixin, DateMixi
   staffPaymentDialog = false
 
   // variables for displaying dialogs
+  courtOrderDialog = false
   resumeErrorDialog = false
-  saveErrorReason: SaveErrorReasons = null
+  saveErrorReason = null as SaveErrorReasons
   paymentErrorDialog = false
+  courtOrderDialogText = 'This business has a court order in its ledger. Please contact BC ' +
+    'Registries to submit a Consent to Continue Out.'
+  courtOrderDialogTitle = 'Please contact BC Registries'
```

Comment:
These were a bit long for the declaration in the template so I created these properties instead.